tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -781,6 +781,9 @@ def _dispatch_home_button_event(self):
     def hold_home_button(self):
         self.marionette.switch_to_frame()
         self.marionette.execute_script("window.wrappedJSObject.dispatchEvent(new Event('holdhome'));")
+        # This is for the opacity animation to be finished for the task-manager
+        # Otherwise we get intermittent issues tapping on opening a new browser window
+        time.sleep(0.3)
```

Comment:
Would it be possible to use a wait done in the Cards View class? Something like:
```py
self.marionette.switch_to_frame()
self.marionette.execute_script("window.wrappedJSObject.dispatchEvent(new Event('holdhome'));")
CardsView(self.marionette).wait_for_card_ready()
```